src/collections/_documentation/enriching-error-data/context.md

```diff
@@ -30,61 +50,64 @@ Users consist of a few key pieces of information which are used to construct a u
--   Summarized within an event on the sidebar.
--   The tags page on an aggregated event.
+As mentioned, tags are searchable key/value pairs attached to events that are used in several places: filtering issues, quickly accessing related events, seeing the tag distribution for a set of events, etc. Common uses for tags include hostname, platform version, and user language.
 
-We’ll automatically index all tags for an event, as well as the frequency and the last time a value has been seen. Even more so, we keep track of the number of distinct tags, and can assist in you determining hotspots for various issues.
+Most SDKs support configuring tags by configuring the scope:
 
-## Setting the Level
+{% include components/platform_content.html content_dir='set-tag' %}
 
-You can set the severity of an event to one of five values: `fatal`, `error`, `warning`, `info`, and `debug`. `error` is the default, `fatal` is the most severe, and `debug` is the least.
+Sentry promotes several pieces of predefined data to tags. We strongly recommend you don't set custom tags with these reserved names. But if you do, you can search for them with a [special tag syntax]({%- link _documentation/workflow/search.md -%}).
 
-{% include components/platform_content.html content_dir='set-level' %}
+### Context
 
-## Setting the Fingerprint
+Custom contexts allow you to attach arbitrary data (strings, lists, dictionaries) to an event. These are unsearchable, but are viewable in the "Additional Data" section of an event.
 
-Sentry uses one or more "fingerprints" to decide how to group errors into issues. More information can be found [here]({%- link _documentation/data-management/event-grouping/index.md -%}).
+![Content%20Freeze%20DON%20T%20add%20content%20Docs%20Additional%20D/Screenshot_2020-03-28_19.36.34.png](Content%20Freeze%20DON%20T%20add%20content%20Docs%20Additional%20D/Screenshot_2020-03-28_19.36.34.png)
 
-## Extra Context
+{% include components/platform_content.html content_dir='set-extra' %}
 
-In addition to the structured context that Sentry understands, you can send arbitrary key/value pairs of data which will be stored alongside the event. These are not indexed and are simply used to add additional information about what might be happening:
+In the past, custom context was called "extra" and set via a method like setExtra(), which is deprecated.
 
-{% include components/platform_content.html content_dir='set-extra' %}
+### Context Size Limits (TO REVIEW)
+
+Maximum payload size: There are times when you may want to send the whole application state as extra data. Sentry doesn't recommend this as the application state can be extensive and easily exceed the 200kB maximum that Sentry has on individual event payloads. When this happens, you’ll get an `HTTP Error 413 Payload Too Large` message as the server response (when `keepalive: true` is set as `fetch` parameter), or the request will stay in the `pending` state forever (for example, in Chrome).
```

Comment:
@untitaker @bruno-garcia these 2 paragraphs say somewhat different things about what happens to large payloads. do you know how we treat it, and/or how we should rephrase this section?